airflow/migrations/versions/0114_2_4_0_add_dataset_model.py

```diff
@@ -55,7 +55,7 @@ def _create_dataset_table():
             ),
             nullable=False,
         ),
-        sa.Column('extra', ExtendedJSON, nullable=True),
+        sa.Column('extra', sqlalchemy_jsonfield.JSONField(json=json), nullable=True),
```

Comment:
since it seems the field is alway spopulated, should we just make it non-nullable?

Reply:
I think you mean the `{}` in the REST API? That's just because of the JsonObjectField used. It returns `{}` if the value is null and I think it's ok. 
The field is still nullable at the DB level, just empty dictionary at the REST API. I can work the REST API to return None if it's preferred?

Reply:
no i'm fine with it as is.  but i actually thought i recalled that, when it was ExtendedJSON it ended up storing None as {} anyway.  but maybe i was wrong about that.

Reply:
defaulted to empty dict now with not nullable